src/locales/es.json

```
@@ -281,6 +281,7 @@
   "The import orders has been uploaded successfully": "Los pedidos de importación se han cargado exitosamente",
   "The timezone you select is used to ensure automations you schedule are always accurate to the time you select.": "La zona horaria que seleccione se utiliza para garantizar que las automatizaciones que programe siempre sean precisas para la hora que seleccione.",
   "There are no packed shipments. Failed to ship order.": "No hay envíos empacados. No se pudo enviar el pedido.",
+  "There are no packed orders at this facility": "No hay pedidos empaquetados en esta instalación",
```

Just add the text in english and don't translate it in specific language
```suggestion
  "There are no packed orders at this facility": "There are no packed orders at this facility",
```